cdap-app-templates/cdap-etl/cdap-data-pipeline/src/main/java/co/cask/cdap/datapipeline/SmartWorkflow.java

```diff
@@ -71,6 +72,8 @@
   public static final String NAME = "DataPipelineWorkflow";
   public static final String DESCRIPTION = "Data Pipeline Workflow";
   private static final Logger LOG = LoggerFactory.getLogger(SmartWorkflow.class);
+  private static final Logger WRAPPERLOGGER = new LocationAwareMDCWrapperLogger(LOG, Constants.EVENT_TYPE_TAG,
```

Comment:
Kept the key name as generic EVENT_TYPE_TAG, since the value can be lifecycle, beforeStageX, afterStageX etc. otherwise we will require separate keys for all these events. Let me know if this make sense